Arpeggiator.cpp

```diff
@@ -0,0 +1,48 @@
+#include <iostream>
+#include "Arpeggiator.h"
+Arpeggiator::Arpeggiator()
+{
+    
+}
+
+Arpeggiator::~Arpeggiator()
+{
+    std::cout << "Goodbye from Arpeggiator Destructor" << std::endl;
+}
+
+std::string Arpeggiator::getArpMode()
+{
+    arpMode =  (arpModeControlValue == 1) ? "Up " : "Down";
+    return arpMode;
+}
+
+void Arpeggiator::displayArpMode()
+{
+    std::cout << "Arpeggiator mode = " << this->getArpMode() << " - Arp mode control value = " << this->arpModeControlValue << std::endl;
+}
+
+int Arpeggiator::modulateChordType(int chordCtrlValue, int chordModInputVal)
+{
+    int chordIdentifier = chordCtrlValue + chordModInputVal;
+    return chordIdentifier;
```

Comment:
no need for the local variable
`return chordCtrlValue + chordModInputVal;` is all you need to write
- [ ] revise